.eslintrc.json

```diff
@@ -101,7 +101,7 @@
             "error",
             "always"
         ],
-        "sort-keys": "warn",
+        "sort-keys": ["warn", "asc", {"caseSensitive": true, "natural": false}],
```

Comment:
what does 'natural' mean here?

Reply:
That one is more to be explicit as that is the default but it is documented here https://eslint.org/docs/rules/sort-keys

Basically it means alphnumeric sorting but don't fix numbers. For example 10 < 5 when natural is false